Explains Towner houses to Towners

Soproxi opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

## About The Pull Request

Adds an extra line to the Towner greet message, saying that you can claim a house as your own by clicking on it.

## Why It's Good For The Game

Addresses Issue #1536 .
This should really be written out somewhere, because it isn't in any documentation, or anything to my knowledge. This is a simple little fix until a future rework of Towners/families/houses etc.

## Proof of Testing (Required)

![image](https://github.com/user-attachments/assets/deeb4a30-4479-480b-b55a-0aa234a74caf)

Red-Honk commented:
--somedae, but for now, you'll just have to live here, probably somewhere your house lies, no better way to find out which than to check in your key under the sleeve works on one of them.

Soproxi commented:
Changed to
"You own one of the decrepit houses around here... you think you remember stashing a spare key just under the door."

Seems more immersive, I suppose!

And updated the old use of day to 'dae'!

RadiantFlash7 commented:
Seems fine enough.

Actuallygoinginsanern commented:
I dont think something like this exactly requires immersive wording, if its for newer players to realize they can get a house, being clear about it is best.
